students/jimin/users/views.py

```diff
@@ -60,17 +62,18 @@ def post(self, request):
 class SignInView(View):
     def post(self, request):
         try :
-            data = json.loads(request.body)
+            data     = json.loads(request.body)
+            email    = data['email']
+            password = data['password']
             
             if not User.objects.filter(email=data['email']).exists():
                 return JsonResponse({"message": "INVALID_USER"}, status=401)
-            else :
-                user = User.objects.get(email = data['email'])
-            
-            if user.password != data['password']:
-                return JsonResponse({"message": "INVALID_USER"}, status=401)
-            
-            
+        
+            if not bcrypt.checkpw(password.encode('utf-8'), User.objects.get(email=data['email']).password.encode('utf-8')):
+                return JsonResponse({"message": "INVALID_PASSWORD"}, status=401)
+
+            token = jwt.encode({'email':email}, SECRET_KEY, algorithm='HS256')
```

Comment:
1. token에 담길 정보로 email 보다는 user data를 정확히 특정 지을 수 있는 id가 더 좋습니다.
2. user id를 가져와야 하기 때문에 user 변수에 User 객체를 따로 담아주면 좋을 것 같습니다.
3. token까지는 잘 만들었습니다. 만든 token을 front에게 응답으로 전달해 주세요.
4. 해당 코드는 무언가를 db에 생성하는 것이 아니기 때문에 상태코드는 200이 적절할 것 같습니다.
```suggestion
            user = User.objects.get(email=email)
            
            if not bcrypt.checkpw(password.encode('utf-8'), user.password.encode('utf-8')):
                return JsonResponse({"message": "INVALID_PASSWORD"}, status=401)

            token = jwt.encode({'user_id' : user.id}, SECRET_KEY, algorithm='HS256')
            return JsonResponse({'access_token' : token}, status=200)
```